x-pack/plugins/maps/common/get_centroid_features.ts

```diff
@@ -21,19 +21,15 @@ import turfArea from '@turf/area';
 import turfCenterOfMass from '@turf/center-of-mass';
 import turfLength from '@turf/length';
 import { lineString, polygon } from '@turf/helpers';
-import {
-  GEO_JSON_TYPE,
-  KBN_IS_CENTROID_FEATURE,
-  KBN_TOO_MANY_FEATURES_PROPERTY,
-} from './constants';
+import { GEO_JSON_TYPE, KBN_IS_CENTROID_FEATURE, KBN_METADATA_FEATURE } from './constants';
 
 export function getCentroidFeatures(featureCollection: FeatureCollection): Feature[] {
   const centroids = [];
   for (let i = 0; i < featureCollection.features.length; i++) {
     const feature = featureCollection.features[i];
 
     // do not add centroid for kibana added features
-    if (feature.properties?.[KBN_TOO_MANY_FEATURES_PROPERTY]) {
```

Comment:
Can you replace the "should not create centroid for too many features polygon" test case with one for KBN_METADATA_FEATURE?